lib/controllers/authors.js

```diff
@@ -0,0 +1,12 @@
+const { Router } = require('express');
+const Author = require('../models/Author');
+
+module.exports = Router()
+  .get('/:id', async (req, res) => {
+    const author = await Author.getAuthorById(req.params.id);
```

Comment:
generally you'd probably wanna write it like ```Author.getById()``` as the author is implied by the fact that it's a method of the Author class